File: if else/evenodd.cpp
/*write a program to check whether a number entered by user is even or odd*/

#include <iostream>
using namespace std;

int main(){
    int num;
    cout<<"Enter any num:";
    cin>>num;
    
    if(num%2==0){
        cout<<"It is even number";
    }
    
    else{
        cout<<"It is odd number";
    }
}

File: if else/evenodd.java
/*#write a program to check whether a number entered by user is even or odd*/

import java.util.Scanner;

class EvenOdd {
    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number:");
        int num = sc.nextInt();

        if (num % 2 == 0) {
            System.out.println(num + " is an even number");
        } else {
            System.out.println(num + " is an odd number");
        }
    }
}

